controllers/etcd_controller.go

```diff
@@ -215,7 +215,27 @@ func (r *EtcdReconciler) reconcile(ctx context.Context, etcd *druidv1alpha1.Etcd
 		}
 	}
 
-	etcd, err := r.updateEtcdStatusAsNotReady(etcd)
+	ss, err := r.getStatefulSet(ctx, etcd)
+	if err != nil {
+		logger.Error(err, "error getting statefulset")
+		return ctrl.Result{
+			Requeue: true,
+		}, err
+	}
+
+	pvcList := &v1.PersistentVolumeClaimList{}
+	if ss != nil {
+		pvcList, err = r.listPersistentVolumeClaims(ss)
+		if err != nil {
+			return ctrl.Result{
```

Comment:
Shouldn't we use `RequeueAfter` here to allow for the statefulset controller to take action?